nvme_gw.config

```diff
@@ -17,8 +17,8 @@ grpc_server_max_workers = 10
 
 [ceph]
 
-ceph_pool = rbd
-ceph_config_file = /etc/ceph/ceph.conf
+pool = rbd
+config_file = /etc/ceph/ceph.conf
```

Comment:
We can squash on rebase/merge. Either through GitHub or manually.

Reply:
@sskaur If you had plans to squash, then sure, no problem with this. I thought you wanted to have several commits, which would be also fine if there were no problems with them like the above one.

Reply:
We can proceed with squashing for this PR.

Reply:
@sskaur Just in case you are not aware. You may squash it yourself, using the interactive git rebase (`git rebase -i HEAD~6`) and then pushing with the force flag.